guides/common/modules/con_prerequisites-for-amazon-ec2-provisioning.adoc

```diff
@@ -6,4 +6,4 @@ The requirements for Amazon EC2 provisioning include:
 * A {SmartProxyServer} managing a network in your EC2 environment.
 Use a Virtual Private Cloud (VPC) to ensure a secure network between the hosts and {SmartProxyServer}.
 * An Amazon Machine Image (AMI) for image-based provisioning.
-include::snip_common-compute-resource-prereqs.adoc[]
+include::snip_common-compute-resource-prereqs.adoc[]
```

Comment:
```suggestion
include::snip_common-compute-resource-prerequisites.adoc[]
```
I am unsure if we really use short names for files. If this is valid, you may require to change the same everywhere else.

Reply:
I assume the only change is the file ending? So completely unrelated to your PR? Can you please check locally?

Reply:
I didn't make any actual change to this file actually. I just tried adding the prerequisite here first, didn't like it, moved it to the other file... and the squashed commit now includes this.

So should I rename the snippet to '...-prerequisites' and change the include everywhere?

Reply:
I recommend it. But not necessarily in this PR.

Reply:
I fixed the commit, and I will look into renaming that snippet file in a different PR.